dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -171,7 +171,7 @@ dependent_startup_wait_for=rsyslogd:running
 
 {% if not skip_pcied %}
 [program:pcied]
-command={% if API_VERSION == 3 and 'pcied' not in python2_daemons %}python3 {% else %} python2 {% endif %}/usr/local/bin/pcied
+command= python3 /usr/local/bin/pcied
```

Comment:
>python3 [](http://example.com/codeflow?start=9&length=7)

You can just remove `python3` because there is correct shebang line inside the script.

And remove the blank before after `=`.